CsvParser/csvParser.go

```diff
@@ -85,7 +85,8 @@ func CsvParser(filePath string) [100]incident.DefaultIncident {
 	return incidentArray
 }
 
-func convertDefaultIncidentToIncident(arr []incident.DefaultIncident) []incident.Incident {
+//* This function takes in an array of 100 default incidents because that is what is provided by the csvParser above *//
+func convertDefaultIncidentToIncident(arr [100]incident.DefaultIncident) []incident.Incident {
```

Comment:
Is there a benefit limiting to 100 incidents here versus when you call it (for example, in csvParser_test.go). Is there ever a case where we might need more than 100 incidents?

Reply:
This is a demo hack for this semester. We didn't want to want to process many more incidents than we thought we were going to use.